requirements.txt

```diff
@@ -1,21 +1,21 @@
 beautifulsoup4
-celery
+celery==4.1.0
 Django==1.10
 django-debug-toolbar
 django-redis
 django-mptt==0.8.7
 django-widget-tweaks
 ipython
-openpyxl==2.4.2
-pillow
-psycopg2-binary
+openpyxl==2.5.2
+Pillow==5.1.0
+psycopg2-binary==2.7.4
 requests
-selenium
-selenium-requests
-tblib
-unidecode
-ua-parser==0.7.3
-user-agents==1.0.1
+selenium==3.11.0
+selenium-requests==1.3
+tblib==1.3.2
+Unidecode==1.0.22
+ua-parser==0.8.0
+user-agents==1.1.0
 sorl-thumbnail==12.4a1
 https://github.com/selwin/django-user_agents/archive/master.zip
 https://github.com/fidals/refarm-site/archive/0.1.6.zip
```

Comment:
You have to bump the refarm-site version to [0.1.7](https://github.com/fidals/refarm-site/releases/tag/0.1.7), isn't it?

Reply:
@artemiy312 , nope, 0.1.7 was created after this code